include/godot_cpp/classes/wrapped.hpp

```diff
@@ -94,80 +94,67 @@ struct Creator<T, typename std::enable_if<std::is_base_of_v<godot::Wrapped, T>>:
+		m_inherits::initialize_class();                                                                                                \
+		if (m_class::_get_bind_methods() != m_inherits::_get_bind_methods()) {                                                         \
+			_bind_methods();                                                                                                           \
+			m_inherits::register_virtuals<m_class>();                                                                                  \
+		}                                                                                                                              \
+		initialized = true;                                                                                                            \
+	}                                                                                                                                  \
+                                                                                                                                       \
+	static const char *get_class_static() {                                                                                            \
+		return #m_class;                                                                                                               \
+	}                                                                                                                                  \
+                                                                                                                                       \
+	static const char *get_parent_class_static() {                                                                                     \
+		return #m_inherits;                                                                                                            \
+	}                                                                                                                                  \
+                                                                                                                                       \
+	static GDExtensionClassInstancePtr create(void *data) {                                                                            \
+		return reinterpret_cast<GDExtensionClassInstancePtr>(new m_class);                                                             \
+	}                                                                                                                                  \
+                                                                                                                                       \
+	static void free(void *data, GDExtensionClassInstancePtr ptr) {                                                                    \
+		if (ptr) {                                                                                                                     \
```

Comment:
This doesn't need a null check, `delete` already handles null pointers. This is only needed if you were to manually call the destructor.